backend/alembic/versions/a8b9c0d1e2f3_add_linked_group_id_to_tasks.py

```
@@ -0,0 +1,373 @@
+                        conn.execute(
+                            sa.text(
+                                """
+                                INSERT IGNORE INTO task_knowledge_base_bindings
+                                (task_id, knowledge_base_id, bound_by, bound_at)
+                                VALUES (:task_id, :knowledge_base_id, :bound_by, :bound_at)
+                            """
+                            ),
+                            binding,
+                        )
+                        total_migrated += 1
+                    except Exception as e:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)